### DISCUSSION TITLE
How to get lua value from java

### DISCUSSION THREAD

**Discussion by amsitlab** [gudzpoz/luajava#159] [Q&A]
for example, I have lua code
```
local x = 5 * 5
```

how to get `x` from java and displaying to android textview

**Comment by marad** [comment]
When you use a `local` variable it's only available in the executed chunk of code as a name for the value, so you have to return the value right inside the chunk.

I think something like this should work:

```java
Integer x = (Integer)lua.execute("local x = 5*5; return x")[0].toJavaObject()
```
